### ISSUE TITLE
Large Artifact Frequency Nerfed? Make Xenoarch Great Again

### ISSUE THREAD

**Issue by Cisrox** [d3athrow/vgstation13#23263]
#### Description of issue
Large artifacts seem to be less common and almost impossible to find currently in xenoarch

#### Difference between expected and actual behavior
Be able to find atleast 1 or 2 large artifacts consistently every round that lasts atleast 1 hour, only find 1 artifact every 4 or 5 rounds EVEN while using powergaming strats 

#### Steps to reproduce
look for large artifacts
never find any

#### Specific information for locating
LARGE ARTIFACTS

#### Length of time in which bug has been known to occur
A year? I haven't played in a while but I think back in 2016 or 2017 when I used to play they were much more common. some time between then and now they became much less common. xenoarch sucks now

**Comment by Cisrox** [comment]
And if I'm wrong then ban me from the server. I think other people can support my claim that they are much less common now. If anyone knows can you link any related PR's that caused this issue

**Comment by AI-Is-Rogue** [comment]
I agree. About a year ago you could find large artifacts easily in isolated and small digsites about the size of 1-3 tiles, but nowadays those digsites are all carbon or empty, and even the larger sites 9 times out of 10 don't have a large artifact. I do think #20762 could be related to this

**Comment by PJB3005** [comment]
I ran the following SDQL query on the test server:
`SELECT /turf/unsimulated/mineral IN global._block(global._locate(144,147,5), global._locate(328, 305, 5)) WHERE artifact_find`
(basically, select every turf with an artifact on the roid)
![image](https://user-images.githubusercontent.com/8107459/59570225-26621600-9095-11e9-81a7-fb466e1947ea.png)

Then I went back about 2k commits to before the PR @AI-Is-Rogue mentioned...
![image](https://user-images.githubusercontent.com/8107459/59570248-a2f4f480-9095-11e9-9fdd-cd000712b826.png)

So that's not a big difference is it?

Oh, but I actually looked at the artifact generation code:

![image](https://user-images.githubusercontent.com/8107459/59570261-e0598200-9095-11e9-8c57-6fc74cf0c42f.png)

Spot the fuckup.

TL;DR artifact count only changed barely, but artifacts were moved into house & garden digsites only so everybody is skipping them with good reason.

**Comment by Cisrox** [comment]
i am fixing this right now

**Comment by Exxion** [comment]
You shouldn't close it until the PR is merged
Just put "fixes #23263" somewhere in the PR body

**Comment by Cisrox** [comment]
ya i have that in the pr body and sorry my bad

**Comment by Exxion** [comment]
It needs to say those specific words, which it did not
